tests/fixture.rs

```diff
@@ -3,13 +3,15 @@ use std::path::PathBuf;
 pub enum Fixture {
     CFF,
     TTF,
+    MATH,
```

Comment:
These fixtures ceased making sense. One should probably give them better names so that they stay on the same level. Right now what you call `MATH` is also a `CFF` or `TTF` font.